doc/spec_TransferElement.dox

```diff
@@ -213,9 +232,50 @@ This documnent is currently still **INCOMPLETE**!
+    - 12.8.2 This is only possible, if the TransferElement is added by passing a TransferElementAbstractor to TransferGroup::addAccessor().
+  - 12.9 Operations executed via TransferGroup::read() and TransferGroup::write():
+    - 12.9.1 first call preXxx() on all high-level elements (cf. \ref transferElement_B_12_1_2_1 "12.1.2.1"),
+    - 12.9.2 next call the corresponding xxxTransferYyy() on all low-level elements (cf. \ref transferElement_B_12_1_2_2 "12.1.2.2"), and
+    - 12.9.3 finally call postXxx() on all high-level elements.
+  - 12.10 The TransferGroup implements a similar exception handling in TransferGroup::read() and TransferGroup::write() as their TransferElement counterparts. This especially means:
+    - 12.10.1 Before preXxx() is called, the TransferGroup checks all pre-conditions (device opened and operation allowed) to make sure no ChimeraTK::logic_error can occur (other exceptions are anyway not allowed in preXxx).
+      - \anchor transferElement_B_12_10_1_1 12.10.1.1 If any ChimeraTK::runtime_error occurs while checking the pre-conditions, it is immediately passed on to the application.
+      - \anchor transferElement_B_12_10_1_2 12.10.1.2 Any pre-condition which is not met will immediately result in a thrown ChimeraTK::logic_error
+      - 12.10.1.3 The exceptions from \ref transferElement_B_12_10_1_1 "12.10.1.1" and \ref transferElement_B_12_10_1_2 "12.10.1.2" are not prioritised by type. The first detected error throws.
+    - 12.10.2 ChimeraTK::runtime_error exceptions thrown by the low-level elements are propagated to the corresponding high-level elements so they are seen by their postXxx() functions (c.f. \ref transferElement_B_6_3 "6.3")
+    - 12.10.3 postXxx() is called for all high level elements, even if some postXxx() throw exceptions (c.f. \ref transferElement_B_5 "5").
+      - 12.10.3.1 If exceptions have been caught from any postXxx(), an exception is thrown after all of them have been executed.
+      - 12.10.3.2 If more than one exception type is detected, the exception type with the highest priority is thrown:
+        - ChimeraTK::logic_error is thrown before any transfer is actually started, so it implicitly has highest priority.
+        - If a ChimeraTK::runtime_error is thrown by a transfer, it will be passed on to the application.
+        - boost::numeric_cast is lowest priority, since in case of a ChimeraTK::runtime_error the content of the application data buffer is unchanged (see \ref transferElement_B_12_11 "12.11").
+        - boost::thread_interrupted cannot take place inside TransferGroups, since they only occur with AccessMode::wait_for_new_data.
+  - \anchor transferElement_B_12_11 12.11 If a ChimeraTK::runtime_error is thrown, the content of the application buffers of all elements is not changed. (c.f. \ref transferElement_B_6_4 "6.4")
+    - 12.11.1 The TransferGroup calls postRead() of all elements with `updateDataBuffer = false`, if any element has thrown a ChimeraTK::runtime_error.
```

Comment:
12.11.1.1 This means if there has been a runtime_error there cannot be a boost::bad_numeric_cast. On the other hand, if there is a boost::bad_numeric_cast, there cannot be a runtime_error because postXxx() only allowed to re-throw them, not raise them.

Reply:
see C.4.2

Reply:
This is a comment at most. IMO there is already too much about stupid numeric_cast errors in the spec. We want to get rid of them.

Reply:
Ok, we can just let them through and terminate the application. If that's acceptable until they have been removed this is also fine.

Reply:
Let's define that this is the intended behaviour at this point.

Reply:
Agreed.